Seminars/Sem.02/Pract.02/Task07.cpp

```diff
@@ -46,55 +46,49 @@ void inputUsers(System &newSystem)
 
 void printUsersList(System &newSystem)
 {
-    std::ifstream readFromFile(FILENAME1);
+    std::ifstream readFromFile(FILENAME);
     if (!readFromFile.is_open())
     {
         std::cout << "ERROR! The file could not be opened!";
         return;
     }
 
-    int i;
-    for (i = 0; i < USERS; ++i)
+    int count = 0;
+    for (int i = 0; i < USERS; ++i)
     {
         readFromFile >> newSystem.systemOfUsers[i].name
             >> newSystem.systemOfUsers[i].email
             >> newSystem.systemOfUsers[i].password;
+
```

Comment:
Again why you want to read the maximum number of students. What will happen when you try to read from the file the maximum possible size of students and the file has less users saved in it.